server-api/installation.md

```diff
@@ -37,10 +38,12 @@ $ docker-compose run --rm --service-ports api yarn run dev:debug
 
 And debug your app with [Chrome Dev Tools](chrome://inspect).
 
-### Configuration
+### Setting up the database seeder for local development (Configuration)
 
 Change configuration in `config/docker/local-development.json` or `config/docker/local.json` and rerun `docker-compose up --build`.
 
+If both the files do no exist create a `config/local.json` and copy the content of `config/local.example.json` and rerun `docker-compose up --build`.
```

Comment:
The files starting with `local` will never exist because they should not be under version control. See https://github.com/lorenwest/node-config/wiki/Configuration-Files

Reply:
@kdan-96 after thinking about it: Could you add a reference to [node-config's configuration files](https://github.com/lorenwest/node-config/wiki/Configuration-Files) here? I think it's pretty important to explain the order how these configuration files are loaded.

Reply:
Can you explain the requirement a bit more?i didnt get it

Reply:
Added the reference